theano/sandbox/solve.py

```diff
@@ -1,75 +1,7 @@
-from __future__ import absolute_import, print_function, division
+import warnings
+from theano.tensor.slinalg import solve  # noqa
```

Comment:
The noqa doesn't solve Travis' error on missing python 3 compatibility imports. Probable we should go back to explicite whitelisting for flake8.